CHANGELOG.md

```diff
@@ -23,7 +23,8 @@ All notable changes to this project will be documented in this file, in reverse
 
 ### Removed
 
-- Nothing.
+- [#183](https://github.com/zendframework/zend-pimple-config/pull/183)
```

Comment:
please link to skeleton, not to pimple ;-)

Reply:
Good catch. That's what I get when I use your PR as an example :D